includes/class-wcpdf-admin.php

```diff
@@ -724,11 +775,18 @@ public function invoice_number_column_sortable( $columns ) {
 		return $columns;
 	}
 
+	/**
+	 * Add invoice date to order search scope
+	 */
+	public function invoice_date_column_sortable( $columns ) {
+		$columns['pdf_invoice_date'] = 'pdf_invoice_date';
+		return $columns;
+	}
 
 	/**
 	 * Pre WC3.X sorting
 	 */
-	public function pre_get_posts_sort_by_invoice_number( $query ) {
+	public function pre_get_posts_sort_by_invoice_number_and_invoice_date( $query ) {
```

Comment:
Not sure about this function name, maybe would be better to have a general one to allow adding more columns in the future?

eg. `pre_get_posts_sort_by_column`